docs/reference/modules/cluster/allocation_awareness.asciidoc

```diff
@@ -21,7 +21,7 @@ attribute called `rack_id` -- we could use any attribute name.  For example:
 
 [source,sh]
 ----------------------
-./bin/elasticsearch --node.rack_id rack_one <1>
+./bin/elasticsearch -Ees.node.rack_id=rack_one <1>
```

Comment:
Now that it's `-E`, do we actually need to prefix every setting name? It clearly made a ton of sense as a system property, but I feel like it's unnecessary as an Elasticsearch setting (and therefore implied as `es.*`).